apps/meteor/client/components/message/variants/threadPreview/ThreadMessagePreviewBody.tsx

```diff
@@ -10,7 +12,13 @@ type ThreadMessagePreviewBodyProps = {
 };
 
 const ThreadMessagePreviewBody = ({ message }: ThreadMessagePreviewBodyProps): ReactElement | null => {
-	const parsedMessage = parseMessageTextToAstMarkdown(message, { colors: true, emoticons: true });
+	const t = useTranslation();
+	const isEncryptedMessage = isE2EEMessage(message);
+
+	const parsedMessage = parseMessageTextToAstMarkdown(
+		message.e2e === 'done' || !isEncryptedMessage ? message : { ...message, msg: t('E2E_message_encrypted_placeholder') },
+		{ colors: true, emoticons: true },
+	);
```

Comment:
Instead of using the parsed message of a static message (`E2E_message_encrypted_placeholder` ) that will always result in the same thing I suggest you check if it's an e2e and if it's encrypted outside the function and if it is, return the string, no the parsed Message. You can change the return type of the component if needed